ra/ra.go

```diff
@@ -1994,13 +2010,7 @@ func (ra *RegistrationAuthorityImpl) PerformValidation(
 		if prob != nil {
 			challenge.Status = core.StatusInvalid
 			challenge.Error = prob
-
-			// TODO(#5545): Spending can be async until key-value rate limits
-			// are authoritative. This saves us from adding latency to each
-			// request. Goroutines spun out below will respect a context
-			// deadline set by the ratelimits package and cannot be prematurely
-			// canceled by the requester.
-			go ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier.Value)
+			ra.countFailedValidation(vaCtx, authz.RegistrationID, authz.Identifier.Value)
```

Comment:
Since this is still "best effort" and never resulting in a user-surfaced error, why not leave it in a goroutine?